common/app/views/fragments/commercial/cardLogo.scala.html

```diff
@@ -0,0 +1,10 @@
+@(branding: common.commercial.Branding, isStandardSizeCard: Boolean)(implicit request: RequestHeader)
+
+<div class="badge @if(!isStandardSizeCard) {badge--branded}">
+    @branding.label
+    <a class="badge__link" href="@branding.sponsorLink" data-link-name="logo link">
+        <img src="@branding.sponsorLogo.url"
+             alt="@{branding.sponsorName}'s logo"
```

Comment:
You can remove the `'s logo` part. The point of an alternative text is not to describe the image but to describe _the content_ of the image.